### PR TITLE
bpo-27428: Fix WindowsRegistryFinder documentation

### PR DESCRIPTION
**Pull request by Himanshu-Lakhara**
Update documentation to clarify WindowsRegistryFinder
implement MetaPathFinder

https://bugs.python.org/issue27428

### CODE REVIEW

**Comment by brettcannon** [comment]
Thanks for the patch, @Himanshu-Lakhara ! I'll try to review this sometime in the next week or two.

**Comment by Himanshu-Lakhara** [comment]
Thanks for reviewing @brettcannon and Sorry for delayed response. Since this is my first cpython PR and first PR on GitHub so I request you to please be thorough. I have previous experience contributing to open source project but not on GitHub. So It is possible that I might have missed something. 

Thanks again for your time. I'll try to be more responsive.

**Comment by brettcannon** [comment]
@Himanshu-Lakhara no worries. I've been very busy as of late so I have not gotten around to doing a review anyway (although I cursory look seems good 😃 ).

**Review by brettcannon — APPROVED** [review]

**Comment by brettcannon** [comment]
I tweaked the news entry, but otherwise LGTM! Once CI passes again I will merge this.

**Comment by Himanshu-Lakhara** [comment]
Thanks @brettcannon for merging and tweaking news entry. It was my first PR to cpython and first PR on GitHub. Learned cpython workflow and Feeling more confident to take up some coding task. 

Thanks again for your guidance.

**Pull request merged**
